grails-app/conf/application.yml

```diff
@@ -117,7 +117,7 @@ grails:
         copyright: ""
         footer: ""
     mail:
-        enabled: false
+        enabled: true
```

Comment:
accidentally pushed

Reply:
Oh yeah, let's remove that when you get a chance.